content/applications/productivity/iot/config/updating_iot.rst

```diff
@@ -0,0 +1,133 @@
+and, lastly a new image can be flashed (physical :abbr:`IoT (Internet of Things)` box). This
+document will explore the various ways to update the :abbr:`IoT (Internet of Things)` box to ensure
+smooth operation of :abbr:`IoT (Internet of Things)` box processes and devices.
+
+Flashing the SD card on IoT box
+===============================
+
+.. important::
+   This update does not apply to the Windows :abbr:`IoT (Internet of Things)` box (Odoo 16 and
+   higher). To update the Windows :abbr:`IoT (Internet of Things)`, first, uninstall the previous
+   version of the Odoo Windows program and then reinstall it, with the most up-to-date installation
+   package. To begin the installation, navigate to the Odoo 16 or higher installation package for
+   Enterprise or Community - Windows edition at `Odoo’s download page <https://odoo.com/download>`_.
+
+In some circumstances, the :abbr:`IoT (Internet of Things)` box's micro SD Card may need to be
+re-flashed with Etcher software to benefit from Odoo's latest :abbr:`IoT (Internet of Things)` image
+update. This means that the Odoo :abbr:`IoT (Internet of Things)` box software may need to be
+updated in instances of a new :abbr:`IoT (Internet of Things)` box, or when a handlers update and
+update from the :abbr:`IoT (Internet of Things)` box home page does not fix issues.
+
+.. warning::
+   It is often imperative to re-flash the :abbr:`IoT (Internet of Things)` box's image after
+   upgrading the Odoo database to a new version.
+
+.. note::
+   A computer with a micro SD card reader/adapter is required to re-flash the micro SD card.
+
+Navigate to Balena's website and download `Etcher <https://www.balena.io/>`_. It is a free and
+open source utility used for burning image files onto drives. Click to `download
+<https://www.balena.io/etcher#download-etcher>`_. Install and launch the program on the computer.
+
+Then download the latest IoT image from `nightly <http://nightly.odoo.com/master/iotbox/>`_ :
+`iotbox-latest.zip`
+
+This image is compatible with all supported versions of Odoo.
+
+The images should be downloaded and extracted to a convenient file location.
+
+After this step is complete, insert the :abbr:`IoT (Internet of Things)` box's micro SD card into
+the computer or reader. Open *Etcher* and select :guilabel:`Flash from file`, then find and select
+the image just downloaded and extracted. Next, select the drive the image should be burned to.
+Lastly, click on :guilabel:`Flash` and wait for the process to finish.
+
+.. image:: updating_iot/etcher-app.png
+   :align: center
+   :alt: Balena's Etcher software dashboard.
+
+.. tip::
+   Balena's Etcher software also allows for the administrator to flash the :abbr:`SD (Secure
+   Digital)` card from an :abbr:`URL (Uniform Resource Locator)`. To flash from an :abbr:`URL
+   (Uniform Resource Locator)`, simply click :guilabel:`Flash from URL` instead of :guilabel:`Flash
```

Comment:
```suggestion
   Digital)` card from a :abbr:`URL (Uniform Resource Locator)`. To flash from a :abbr:`URL
   (Uniform Resource Locator)`, simply click :guilabel:`Flash from URL` instead of :guilabel:`Flash
```